examples/simple-lambda-in-vpc/example.tf

```diff
@@ -20,6 +20,7 @@ module "lambda_api" {
   vpc_id                        = module.acs.vpc.id
   public_subnet_ids             = module.acs.public_subnet_ids
   role_permissions_boundary_arn = module.acs.role_permissions_boundary.arn
+  xray_enabled                  = true
```

Comment:
(nit) I would remove this flag from one of the simple examples.

Reply:
Done. I did it from both.